app/services/users.js

```diff
@@ -18,6 +20,10 @@ exports.createUser = async (userInfo, roleCode = regRoleCode) => {
 
   const createdUser = (await userModel.create(userInfoWithRole)).toJSON();
 
+  emailHelper.sendMail(email, userCreated.subject, userCreated.message).catch(error => {
```

Comment:
Call this method in the controller and please don't mix promises and async / await